consumer/pdata/trace.go

```diff
@@ -130,29 +130,69 @@ const (
 	SpanKindCONSUMER    = SpanKind(otlptrace.Span_SPAN_KIND_CONSUMER)
 )
 
+// DeprecatedStatusCode is the deprecated status code used previously.
+// https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-status
+type DeprecatedStatusCode otlptrace.Status_DeprecatedStatusCode
```

Comment:
Consider to also mark the field deprecated.

Reply:
Done.